.gitignore

```diff
@@ -4,3 +4,6 @@
 *.log
 *.gz
 *.pdf
+!examples/*.pdf
+.~lock*
+*.out
```

Comment:
Following the same idea, we probably don't want the exception in allowing pdfs; perhaps update the .yaml file so it'll get build on merging the PR?

Reply:
Can I be of help here?

Reply:
Is the resulting PDF up to your standards? Perhaps looking through the .tex file to see if you like how it's written (imagine your were writing it, is there anything odd/difficult to read).

The CI and including PDFs is more @yjyao area right now. I just pointed this out due to early comments of theirs.

Reply:
Not as of yet.

1. The Party names on the title page should be the same font size as the rest of the text.

2. There should be more space within the tramlines

3. The left alignment on the first line is inconsistent.

4. The 'in each case...' subparagraph should be left aligned with the quotation marks in "Data Transfer" in para 1.1.8

<img width="549" alt="image" src="https://user-images.githubusercontent.com/69139574/107998270-065ec480-6fe5-11eb-89cc-438729521d3e.png">

5. Same thing with 'The terms, “Commission”, [...]' should be aligned with "Unless" in 1.1

6. Same thing with 6.2. The subparagraph should be aligned with "Processor shall".

7. 12.1.0.1 should be 12.1.1., if I am not mistaken?

Thank you so much, looks great otherwise!

Reply:
i'm going to move @passamo9's comment to their own threads. please don't reply here and focus on the .gitignore on this thread.

and yes please remove `!example/*.pdf`.

Reply:
So sorry @yjyao. Thank you for cleaning up after me!

Reply:
please don't say sorry. i didn't mean to micromanage but i think it'd be easier to keep conversations to their threads (which means i shouldn't be replying right now). right now we have too many threads discussing too many things and it's becoming hard to track.

Reply:
I may have donked up the .gitignore.  I was just trying to add the libreoffice lock to it so I could read without git giving me back-chat. Feel free to remove anything that's a pain.

Reply:
It'd be easier if you could, as some of us don't have the ability to edit your PR. Though @yjyao or @Giebisch could come to the rescue.

Reply:
@bond7o6 fixed in 36159179bcd03c24